### DISCUSSION TITLE
error on dashboard screen

### DISCUSSION THREAD

**Discussion by Rup16** [nextgenhealthcare/connect#5385] [General]
Method failed: HTTP/1.1 500 Server Error com.mirth.connect.client.core.ClientException: Method failed: HTTP/1.1 500 Server Error at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:533) at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:457) at com.mirth.connect.client.core.ServerConnection.executeAsync(ServerConnection.java:303) at com.mirth.connect.client.core.ServerConnection.apply(ServerConnection.java:169) at org.glassfish.jersey.client.ClientRuntime.invoke(ClientRuntime.java:255) at org.glassfish.jersey.client.JerseyInvocation$3.call(JerseyInvocation.java:722) at org.glassfish.jersey.internal.Errors.process(Errors.java:315) at org.glassfish.jersey.internal.Errors.process(Errors.java:297) at org.glassfish.jersey.internal.Errors.process(Errors.java:228) at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:444) at org.glassfish.jersey.client.JerseyInvocation.invoke(JerseyInvocation.java:718) at org.glassfish.jersey.client.JerseyInvocation$Builder.method(JerseyInvocation.java:430) at org.glassfish.jersey.client.proxy.WebResourceFactory.invoke(WebResourceFactory.java:381) at com.sun.proxy.$Proxy68.getDashboardChannelInfo(Unknown Source) at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) at java.lang.reflect.Method.invoke(Unknown Source) at com.mirth.connect.client.core.Client$2.invoke(Client.java:270) at com.sun.proxy.$Proxy68.getDashboardChannelInfo(Unknown Source) at com.mirth.connect.client.core.Client.getDashboardChannelInfo(Client.java:1221) at com.mirth.connect.client.ui.Frame$12.doInBackground(Frame.java:2422) at com.mirth.connect.client.ui.Frame$12.doInBackground(Frame.java:2405) at com.mirth.connect.client.ui.QueuingSwingWorker.doInBackground(QueuingSwingWorker.java:97) at javax.swing.SwingWorker$1.call(Unknown Source) at java.util.concurrent.FutureTask.run(Unknown Source) at javax.swing.SwingWorker.run(Unknown Source) at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) at java.lang.Thread.run(Unknown Source) Caused by: com.mirth.connect.client.core.EntityException: { "servlet":"org.glassfish.jersey.servlet.ServletContainer-334ed7d4", "message":"Request failed.", "url":"/api/4.1.0/channels/statuses/initial", "status":"500" } at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:530) ... 29 more

**Comment by pacmano1** [comment]
1) Use code blocks when posting
2) Don't post a long error messaage wiithout the operating system and/or if docker, java version, mirth version and any other relevant things you were doing when the error occurred.

While this is not the Mirth  forums, read this https://forums.mirthproject.io/articles/177082-forum-etiquette.   It is a good guide for how to aks for help here and other pulbic forums / discussions.

**Comment by jonbartels** [comment]
Whats in `mirth.log` on the server?

**Comment by rjkroll** [comment]
See this post about your issue.    Seems most everybody upgrading to 4.1 is going to have this error unless they notice that Java 9+ and 4.1 Mirth Connect need additional configuration:  **https://github.com/nextgenhealthcare/connect/discussions/5392**

**Reply by pacmano1** [reply]
That has been in the readme for a
few releases.  People should read product manuals.
